about.html

```diff
@@ -0,0 +1,120 @@
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><button type="button">JSI Conference</button></li>
+            </ul>
+            <div class="mob-menu hidden">
+                <i class="fa fa-times" aria-hidden="true" id="close"></i>
+                <ul class="menu-link">
+                    <li><a href="index.html" class="links">Home</a></li>
+                    <li><a href="#" class="links">Program</a></li>
+                    <li><a href="#" class="links">Join</a></li>
+                    <li><a href="#" class="links">Sponsor</a></li>
+                    <li><a href="#" class="links">News</a></li>
+                    <li><button type="button">JSI Conference</button></li>
+                </ul>
+            </div>
+        </nav>
+ </header>
+ <main>
+     <!-- INTRODUCTION SECTION  -->
+    <section class="about-intro">
+        <div>
+           <h2>"Hello! Sharing world"</h2>
+           <h1>International Javascript<br>Conference 2022</h1>
+        </div>
+        <article>
+         <p class="about-txt">International javascript Conference is a summit that talks about the latest innovations in javascript.<br>This is to help broaden the knowledge of web developers and javascript enthusiast.<br> An open source community that believes in the value of openness, sharing, creativity and tech empowerment.<br> Creating a positive change with javascript enthusiast all over the world. We are hosting the biggest javascript conference.</p>
+         <p class="about-email">Please contact us via email for any further questions</p>
+         <h4>info@ijs.com</h4>
+        </article>
+    </section>
+    <!--  ABOUT LOGO SECTION  -->
+    <section class="about-logo">
+      <div class="a-title">
+         <h3>International Javascript Conference Logo</h3>
+         <p>The logo of the International Javascript conference was decided through the logo competition from 8 August to 7 september</p>
+      </div>
+      <div>
+          <img src="images/logo.png" alt="logo">
+      </div>
+    </section>
+    <hr>
+    <!--   PAST EVENT SECTION   -->
+    <section class="Past-events">
+      <div class="event-title">
+      <h3>See the past International Javascript Conference</h3>
+      <p>Take a look at the last two conference which took place in Hawaii and Bulgaria</p>
+      </div>
+      <div class="event-img">
+          <div class="event-card">
+          <img src="./images/past.jpg" alt="past event picture">
+          <p class="center-text"><span>2018</span><br>international javascript conference in hawaii</p>
+          </div>
+          <div class="event-card">
+          <img src="./images/past2.jpg" alt="past event picture 2">
+          <p class="center-text"><span>2020</span><br>international javascript conference in bulgaria</p>
+          </div>
+      </div>
+    </section>
```

Comment:
- [ ] In this section also there is some difference between the image sizes of yours and the original design. Take a look below 👇 
- Yours 👇 
![see-past-st](https://user-images.githubusercontent.com/67087664/147964869-7d37cabe-9cc4-4fe9-921a-acd6a199277a.png)
- Original 👇 
![see-past](https://user-images.githubusercontent.com/67087664/147964902-e67c1352-3f40-4343-b65b-ffe4c1d44336.png)

Kindly fix it 🙏